keras/utils/data_utils.py

```diff
@@ -186,19 +186,21 @@ def get_file(fname,
 
     if download:
         print('Downloading data from', origin)
-        progbar = None
 
-        def dl_progress(count, block_size, total_size, progbar=None):
-            if progbar is None:
-                progbar = Progbar(total_size)
+        # Closures: Use a dictionary workaround To support python2,
+        # since `nonlocal` is only support in python3.
```

Comment:
Change applied in #6670 in addition to a second bugfix that occurs when servers don't provide a Content-Length header.